### PR TITLE
[formal/doc] Add a getting started document for creating and running formal

### PR DESCRIPTION
**Pull request by cindychip**
This PR adds a getting started document to gather all instructions to
create and run FPV or connectivity tests in OpenTitan.

Signed-off-by: Cindy Chen <chencindy@opentitan.org>

### CODE REVIEW

**Review by rswarbrick — APPROVED** [review]

**Review by weicaiyang — APPROVED** [review]
LGTM

File: doc/ug/getting_started_formal.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+It is recommended to use the [fpvgen]({{< relref "util/fpvgen/README.md" >}}) tool to automatically create a FPV testbench template.
+To run the FPV tests in `dvsim`, please add the target in `hw/top_earlgrey/formal/top_earlgrey_fpv_cfgs.hjson`, then run with command:
+```console
+$ util/dvsim/dvsim.py hw/top_earlgrey/formal/top_earlgrey_fpv_cfgs.hjson --select-cfgs {target_name}
+```
+It is recommended to add the FPV target to [lint]({{< relref "hw/lint/doc/README" >}}) script `hw/top_earlgrey/lint/top_earlgrey_fpv_lint_cfgs.hjson` to quickly find typos.
+
+## Formal connectivity verification
+
+The connectivity verification is mainly used for exhaustively verify system-level connections.
```
Comment (matutem) [inline doc/ug/getting_started_formal.md:21]:
```suggestion
The connectivity verification is mainly used for exhaustively verifying system-level connections.
```

File: doc/ug/getting_started_formal.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+This document aims to enable a contributor to get started with a formal verification effort within the OpenTitan project.
+While most of the focus is on development of a testbench from scratch, it should also be useful to understand how to contribute to an existing effort.
+
+Please refer to the [OpenTitan Assertions]({{< relref "hw/formal/README" >}}) for information on how formal verification is done in OpenTitan.
+
+## Formal property verification (FPV)
+
+It is recommended to use the [fpvgen]({{< relref "util/fpvgen/README.md" >}}) tool to automatically create a FPV testbench template.
```
Comment (matutem) [inline doc/ug/getting_started_formal.md:12]:
```suggestion
It is recommended to use the [fpvgen]({{< relref "util/fpvgen/README.md" >}}) tool to automatically create an FPV testbench template.
```

**Review by matutem — APPROVED** [review]
LGTM with minor suggestions.

File: doc/ug/getting_started_formal.md (hunk 1 above)
Comment (cindychip) [reply doc/ug/getting_started_formal.md:21]: Thanks Matute!

**Pull request merged**